### ISSUE TITLE
'null' album information

### ISSUE THREAD

**Issue by CentreMetre** [moritz-weber/mucke#150]
**Describe the bug**
Ive only just started using this app so sorry if its obvious, also if i used the wrong github issue type.
In the album information it says: null (dot) 13 songs (dot) 35:26
Im not sure what the null is or how to fix it. Is it release year?
I have seen #70 but im not sure there was a solution? Maybe i didnt look properly

**To Reproduce**
Steps to reproduce the behaviour:
1. Imported folder with the album on phone,
2. Added it to the Mucke library
3. Go on the album
4. See null (dot) song amount (dot) total playing time

**Expected behaviour**
For it to not show null but the value that is supposed to be there

**Screenshots**
![image](https://github.com/moritz-weber/mucke/assets/61330791/a2bf2c2f-c8df-49d9-a8e0-84ef1e9f42d3)

**Smartphone (please complete the following information):**
 - Device: Google Pixel 6
 - OS: Graphene OS
 - Version : 2023091800 (i think)

**Additional context**

**Comment by moritz-weber** [comment]
No worries. You're right, this should display the year of the album.
This "basic" metadata stuff should be pretty reliable by now.
Could you check in another app if the songs actually include the information about the year?

Independent of this bug report, we could think about just not displaying anything if this information is missing.

**Comment by CentreMetre** [comment]
I used AIFF files (being a newbie at audio music stuff) thinking it would just work, but I'm guessing its not standard. The fact that AIFF files didn't show in the allowed file extension should've told me that.
Since I also couldn't even play the songs on any player I assumed maybe the AIFF didn't work, so now I swapped to FLACs and now its working, so it was my fault, also the year does show now with the FLACs. Sorry bout that.
